SDNA improvements (#457)
perspect3vism/ad4m: docs/pages/social-dna.mdx

turing complete logical programming language, any kind of alorithm can be run based on the
state of the graph.

<Callout type="info" emoji="ℹ️">

>   Don't worry if this looks very unfamiliar with [Prolog](https://en.wikipedia.org/wiki/Prolog). You will normally either

Reviewer: ```suggestion
  Don't worry if you are unfamiliar with [Prolog](https://en.wikipedia.org/wiki/Prolog). You will normally either
```
